src/language-js/print/array.js

```diff
@@ -75,11 +78,25 @@ function printArray(path, options, print) {
         return element[itemsKey] && element[itemsKey].length > 1;
       });
 
+    const shouldUseFill =
+      n.elements.length > 1 &&
+      n.elements.every(
+        (element) =>
+          element &&
+          (isNumericLiteral(element) ||
+            (isStringLiteral(element) && element.value.length < 9))
```

Comment:
Should  use  `raw` instead of `value`

Reply:
And 8 is  not  enough for color `#RRGGBBAA`

Reply:
Using `raw` caused an error with `espree`. And this change did not work on `estree` or `meriyah`:

```suggestion
            (isStringLiteral(element) && (element.raw || element.value).length < 9))
```

It also did not resolve my issue with cursorOffset & full-width characters in `tests_integration/__tests__/cursor-offset.js`.

Reply:
I mean https://github.com/prettier/prettier/blob/33078c61eb7388afa80a5057968cd656f7b7f0c8/src/language-js/utils.js#L968